internal/current_target.go

```diff
@@ -18,14 +19,22 @@ func LoadCurrentTarget(currentTargeFile string) (CurrentTarget, error) {
 	if err != nil {
 		return cur, err
 	}
-	cur.Name = ctToml.Get("TARGET_NAME").(string)
-	verStr := ctToml.Get("CUSTOM_VERSION").(string)
-	cur.Version, err = strconv.Atoi(verStr)
-	if err != nil {
-		return cur, err
+	if val, ok := ctToml.Get("TARGET_NAME").(string); !ok {
+		return cur, fmt.Errorf("Unable to parse current-target: TARGET_NAME: %s", val)
```

Comment:
IIUC `val` is always an empty string in case if the type conversion fails. So there is no value to print it.

Also, the most probable reason when the type conversion would fail is when the Toml value is missing.
So, I would simply combine two conditions into one to avoid confusion.

Together with simplifying the below code, I think what you really wanted to achieve is this:
```
var ok bool
val := ctToml.Get("TARGET_NAME")
if cur.Name, ok = val.(string); !ok || len(cur.Name) == 0 {
    return cur, fmt.Errorf("Unable to parse current-target: TARGET_NAME: %v", val)
}
```

The same applies to the version parsing code.

...I'm so sorry, as I have a feeling we spend to much on this.